go-controller/pkg/config/config.go

```diff
@@ -470,6 +471,11 @@ var K8sFlags = []cli.Flag{
 		Usage:       "specify a namespace which will contain services to config the OVN databases",
 		Destination: &cliConfig.Kubernetes.OVNConfigNamespace,
 	},
+	cli.StringFlag{
+		Name:        "metrics-bind-address",
+		Usage:       "specify an address for a prometheus metrics server to scrape",
```

Comment:
@JacobTanenbaum let's grab what kube-proxy says here for this option:

`The IP address and port for the metrics server to serve on (set to `0.0.0.0` for all IPv4 interfaces)`

Reply:
Done